pkg/resource/sensitive_test.go

```diff
@@ -227,6 +231,15 @@ func TestGetSensitiveAttributes(t *testing.T) {
 				},
 			},
 		},
+		"Optional": {
```

Comment:
Would you mind adding a case where the field does not exist at all, i.e. not even `null`?

Reply:
We already have a test case for that: https://github.com/crossplane-contrib/terrajet/blob/e1a8ff4a3f6c28c31101a95533c54150c1917631/pkg/resource/sensitive_test.go#L230